--- a/configure
+++ b/configure
@@ -20274,7 +20274,7 @@
       if test `echo $host | grep -c apple-darwin` -ne 0 -a `echo $CURL_LIB | grep -c "^-L"` -gt 0; then
         CURL_LIB_PATH=`echo ${CURL_LIB} | tr " " "\012" | grep "^-L" | tr "\012" " " | sed 's:-L::' | sed 's: $::'`
         if test "x$CURL_LIB_PATH" != x; then
-          LIBS_CURL="-Wl,-rpath,$CURL_LIB_PATH $CURL_LIB"
+          LIBS_CURL="$CURL_LIB"
         fi
       fi
       if test `echo $host | grep -c cygwin` -ne 0 -o `echo $host | grep -c mingw32` -ne 0; then
