diff --git a/longnam.h b/longnam.h
index a304faf..8ce04eb 100644
--- a/longnam.h
+++ b/longnam.h
@@ -19,7 +19,8 @@
    that was used to build the CFITSIO library is compatible with the version
    as included when compiling the application program
 */
-#define fits_open_file(A, B, C, D)  ffopentest( CFITSIO_SONAME, A, B, C, D)
+/*#define fits_open_file(A, B, C, D)  ffopentest( CFITSIO_SONAME, A, B, C, D)*/
+#define fits_open_file  ffopen
 
 #define fits_open_data      ffdopn
 #define fits_open_extlist   ffeopn
